src/components/SecondaryPanes/EventListeners.js

```diff
@@ -97,14 +97,15 @@ class EventListeners extends Component<Props> {
 
 export default connect(
   state => {
-    const listeners = getEventListeners(state).map(l =>
-      Object.assign({}, l, {
+    const listeners = getEventListeners(state).map(l => {
```

Comment:
` getEventListeners(state).map(l => ({})` will work too

Reply:
I was actually perplexed for a while as how to do it, thanks for the follow up!